### ISSUE TITLE
Port number changes

### ISSUE THREAD

**Issue by sslupsky** [arduino-libraries/MKRWAN#29]
I am not sure if this is a bug yet.

While experimenting with the example code, I noticed that the port number changes from time to time.  I have not figured out why it changes yet.  I have seen it change when you reset the MKRWAN1300.  I have also seen what I think is just a random change.  That is, several hours after an uplink I sent another uplink and the port number changed on the new uplink.

I've seen uplinks using port number 2, 4 and 6.  Looking further into this, I cannot find the API for setting the port number for an uplink.

**Comment by sslupsky** [comment]
I found an API call to set the port in MKRWAN.h

bool setPort(uint8_t port)

I am still unsure still why the port number is changing.  At the moment I do not specifically set the port number at any point in the code so I am wondering what method the MKRWAN firmware uses to set the port number?

**Comment by sslupsky** [comment]
This appears to be a mkrwan-fw issue.

https://github.com/arduino-libraries/MKRWAN/issues/8

**Comment by sslupsky** [comment]
I am re-opening this issue.  I programmed a new MKRWAN1300 today and when the sketch makes a call to setPort(10), the port number does not get set correctly.  The server is showing that the messages are being sent on Port 21.  I power cycled the board and now the messages are sent on Port 19.  I reprogrammed the board and now the messages are being sent on Port 20.  

In all cases the messages should be sent on Port 10.
